resources/styles/molecules/accordion/accordion.scss

```diff
@@ -33,28 +33,30 @@
         border-top: 1px solid config.$gray-100;
         padding: config.$thin-v-space 0;
         margin: 0 1rem;
+        max-height: 100vh;
```

Comment:
Animating max-height (because you can't animate to height: auto) is a neat trick, but the end result is unfortunately always kinda janky. There are other pure CSS tricks out there, but this is why we had the JS to support the transition in the BP5 version.

If you want to try a different approach (using JS or not) that can be close to as smooth as the old version, go for it, or you want to remove the transition entirely you can do that too, but animating max-height is worse than no transition, sorry.

Reply:
Mm, Tim found a bug where in _custom-properties.scss the definition of these should be 
```
    --duration-fast: #{config.$fast};
    --duration-moderate: #{config.$moderate};
    --duration-slow: #{config.$slow};
```
I think this might be have been why the original animations stopped working; if you add in this fix I bet you'll be able to get the original animations working.